README.md

```diff
@@ -215,6 +223,204 @@ objectives:
     that is used as the denominator. Received data is used to compare objectives
     (threshold) values to find total number of metrics.
 
+#### Alert Policy
+
+An Alert Policy allows you to define the alert conditions for a SLO.
+
+```yaml
+apiVersion: openslo/v1alpha
+kind: AlertPolicy
+metadata:
+  name: string
+  displayName: string # optional
+spec:
```

Comment:
Another thing you may want to consider adding to the spec: How the _absence_ of data points are handled. E.g. add an `alertIfAbsent` boolean field to the spec (either in `AlertPolicy` or `AlertSpec`).

Reply:
Ah you mean to allow defining an alert when for example the `up`-metric is missing in Prometheus time series? So you can alert when a service, correct?